.github/workflows/benchmarks.yml

```diff
@@ -39,6 +39,7 @@ jobs:
           - hnswlib
           - luceneknn
           - milvus
+          - glass
```

Comment:
can you keep this in alphabetical order? thanks

Reply:
Sure